src/Domain.LinnApps/Reports/ImportBookReportService.cs

```diff
@@ -328,186 +354,197 @@ private void AddExportColumns(SimpleGridLayout reportLayout)
                     {
                         RowId = $"{impbook.Id.ToString()}/{orderDetail.LineNumber}",
                         ColumnId = "ForeignValue",
-                        //todo change this to be value from intercompany inv
-                        TextDisplay = string.Empty,
+                        TextDisplay =
+                            exportReturnDetail != null ? exportReturnDetail.CustomsValue?.ToString() : string.Empty,
                         RowTitle = impbook.Id.ToString()
                     });
         }
 
         private void ExtractExportDetails(
-        ICollection<CalculationValueModel> values,
-        ImportBook impbook,
-        ImportBookOrderDetail orderDetail,
-        bool isInEU = false)
+            ICollection<CalculationValueModel> values,
+            ImportBook impbook,
+            ImportBookOrderDetail orderDetail,
+            bool isInEU)
         {
+            var exportReturnDetail = (ExportReturnDetail)null;
```

Comment:
I don't think I like this, not sure it's the best. Anyone think it'd be better just initialising it normally and then checking the same conditions as the if below, instead of checking for exportReturnDetail being null later on? Or any other suggestions if anyone is bothered

Reply:
I didn't see this comment since github hides larges diffs.. 
Yeah - this seems like an odd thing to do. If you need to initialise exportReturnDetail with a null value it doesn't make much sense to make it an implicitly typed var

Reply:
I don't know what I was thinking with this yesterday, I'll just change var to ExportDetailReturn and set it to null in my next impbook pr